src/org/pursuit/Main.java

```diff
@@ -212,19 +196,27 @@ private void viewHomePage() {
 
     }
 
-    private Integer getAccountByCustomerName() {
```

Comment:
This is one of the bigger pain points to be mindful of. Remember that when writing a method, naming is always really important. In calling a method getAccount...(), it seems that method will return an Account object. Instead this states it returns Integer. And then similarly, getAccountByCustomerName() looks like you're searching for said account by the Customer's name, which would be an input to the method. But there are no parameters here.